src/hashTable.js

```diff
@@ -0,0 +1,60 @@
+export default class HashTable {
+  constructor() {
+    this.storage = []
+    this._length = 0
+    this.max = 10
+  }
+
+  put = (key, value) => {
+    if(typeof(key) === "undefined"){
+      throw("Cannot insert with undefined key!")
+    }
+    const hashIndex = this.hash(key, this.max)
```

Comment:
See below - you probably want to be storing the key as well as the value, and using separate chaining.